examples/violations-monorepo/packages/business-logic/.structlintrc

```diff
@@ -0,0 +1,28 @@
+{
+  "structure": [
+    {
+      "description": "Business Logic",
+      "path": "./",
+      "disallowedImports": [
+        {
+          "glob": "@monorepo/mobile-app",
+          "message": "Don't import mobile app into business-logic"
+        },{
+          "glob": "@monorepo/web-app",
+          "message": "Don't import web app into business-logic"
+        }, {
+          "glob": "@monorepo/components",
+          "message": "Don't import components into business-logic"
+        },
+        {
+          "glob": "../components/**/*",
```

Comment:
hm, so maybe we should read all `package.json` and disallow relative imports together with package imports (@monore/components)?

Reply:
yes. But before that let's implement https://github.com/desfero/structlint/issues/2 so that if `structlint` is run in a directory where the `structlintrc` file doesn't have a `root` property, it won't look for workspaces.